.settings.sh

```diff
@@ -0,0 +1,15 @@
+if [[ ! -x vault-password ]]
+then
+  echo "./vault-password does not exist, please read README.md"
+else
+  export ANSIBLE_VAULT_PASS="$($(pwd)/vault-password)"
+
+  if [[ -r settings.vault ]]
+  then
+    echo "Sourcing ./settings.vault"
+    $(ansible-vault view settings.vault)
+  fi
```

Comment:
Burde den si fra om `settings.vault` mangler? som en ikke-fatal-feil?

Reply:
Kanskje, kanskje ikke. Enn så lenge så er den for de som har ekstra mye tilgang via Linode som trenger den. Tenkte at det var greit å ikke spamme alle, men i praksis så har vel kanskje alle som gjør noe her også tilgang til Linode og da kan en advarsel være på plass.